plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/PostgreUtils.java

```diff
@@ -223,8 +215,8 @@ public static long[] getIdVector(Object pgObject) {
         }
     }
 
-    public static int[] getIntVector(Object pgObject) {
-        Object pgVector = extractPGObjectValue(pgObject);
+    public static int[] getIntVector(@Nullable Object pgObject, @Nullable DBPDataSource dataSource) {
```

Comment:
How can dataSource be null?
It shouldn't because objects exist only within a live connection.,